packages/genesys-spark-components/src/components/beta/gux-avatar-focusable/gux-avatar-focusable.light.scss

```diff
@@ -0,0 +1,15 @@
+@use '../../../style/_focus.scss';
+
+gux-avatar-focusable-beta {
+  a {
+    border-radius: 50%;
+  }
+
+  &.gux-size-small,
+  &.gux-size-xsmall {
+    a:focus-visible,
+    button:focus-visible {
+      @include focus.gux-focus-ring-small;
+    }
+  }
+}
```

Comment:
I think it would be safer to do all the styling in the light dom. Is the a reason we shouldn't do that?

Reply:
I was wondering the same thing when I was updating it. I wasn't sure if I should only have the styles that need to be in the light dom in the light dom file. Updated now